### ISSUE TITLE
_escaped_fragment_ is deprecated by google 3 years ago

### ISSUE THREAD

**Issue by yanchogeorgiev** [jeroennoten/Laravel-Prerender#22]
https://webmasters.googleblog.com/2015/10/deprecating-our-ajax-crawling-scheme.html
Is this library also deprecated?

**Comment by thoop** [comment]
No, this library is not deprecated. This library checks googlebot by their user agent directly instead of following the escaped fragment protocol:

https://github.com/jeroennoten/Laravel-Prerender/blob/master/config/prerender.php#L144

**Comment by butaminas** [comment]
Just for the record and to make this more clear for everyone else, if I enable 'googlebot' in 'crawler_user_agents' list (using prerender.io), am I at risk to being penalized for cloaking?
Is there any reason why I shouldn't allow googlebot, yahoo, and bingbot while using prerender.io?

**Comment by thoop** [comment]
If you enable 'googlebot' in the list of user agents, you are not at risk of being penalized for cloaking.
Checking Googlebot by their user agent to return a prerendered page follows Google's Dynamic Rendering specifications:

https://developers.google.com/search/docs/guides/dynamic-rendering

Google even recommends using Prerender.io in the documentation on Dynamic Rendering.

**Comment by denjaland** [comment]
Hiya,

I just installed the package, but in the published config file, I literally read the following:

     'crawler_user_agents' => [
        // googlebot, yahoo, and bingbot are not in this list because
        // we support _escaped_fragment_ and want to ensure people aren't
        // penalized for cloaking.

        // 'googlebot',
        // 'yahoo',
        // 'bingbot',

So are we really safe to uncommend those three?
I notice the change in config/prerender to date from AFTER the last release, so I'm still getting the conig with the above message.

Is this library actively maintained still?

**Comment by victorsilent** [comment]
@thoop, same thing as @denjaland
